Base/QTGUI/qSlicerMouseModeToolBar.cxx

```diff
@@ -172,6 +172,17 @@ void qSlicerMouseModeToolBarPrivate::init()
   q->addAction(this->PlaceWidgetAction);
 
   q->addAction(this->ToolBarAction);  // add Toggle Markups ToolBar action last
+
+  this->PlaceWidgetToolBarAction = new QAction(this);
```

Comment:
Could you add a comment here and also above `this->PlaceWidgetAction = new QAction(this);` to explain where each action is being used  ?

Reply:
`PlaceWidgetToolBarAction` is in the menu of the Place widget and that `ToolBarAction` is an action represented by a QToolButton in the Markups Toolbar?

Is that level of description clear for you? If so I'll add those comments.

Reply:
@jcfr Those docstrings have now been added.